addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -1579,7 +1579,7 @@ def test_in_invoice_payment_register_wizard(self):
         self.assertFalse(move.payment_ids)  # don't auto reconcile payments
 
         # If the move is already fully paid, we should alert the user
-        with self.assertRaisesRegex(UserError, r"You can't register a payment because there is nothing left"):
+        with self.assertRaisesRegex(UserError, r"There's nothing left to pay on those selected journal items, so no payment registration is necessary. You've got your finances under control like a boss!"):
```

Comment:
don't have the full context to definitely write this correctly, but I think it's supposed to be:

```suggestion
        with self.assertRaisesRegex(UserError, r"There's nothing left to pay for the selected journal items, so no payment registration is necessary. You've got your finances under control like a boss!"):
```